src/main/java/appeng/util/Platform.java

```diff
@@ -803,7 +805,20 @@ public static boolean isWrench( final EntityPlayer player, final ItemStack eq, f
 			{ // explodes without BC
 
 			}
-
+			
+			// CoFH Wrench Support
+            try {
+                if( eq.getItem() instanceof IToolHammer )
+                {
+                    IToolHammer wrench = (IToolHammer) eq.getItem();
+                    return wrench.isUsable( eq, player, pos );
+                }
+            }
+            catch( final Throwable ignore )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1166)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1181)